src/Microsoft.Build.Sql/sdk/Sdk.props

```diff
@@ -26,10 +28,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">16.0</VisualStudioVersion>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">17.0</VisualStudioVersion>
     <!-- Default to the v16.0 targets path if the targets file for the current VS version is not found -->
```

Comment:
Update comment?

Reply:
@ErikEJ Looks like you are reviewing code from a single commit only. Comment is updated as well

Reply:
Yeah, missed that